File: 프로그래머스/unrated/181855. 문자열 묶기/문자열 묶기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<string> strArr) {
    int answer = 0;
    int nums[100000] = {0}; // 길이 저장 배열
    for(string str : strArr){
        nums[str.size()]++;
    }
    
    return *max_element(begin(nums), end(nums));
}

File: 프로그래머스/unrated/181855. 문자열 묶기/문자열 묶기.java
import java.util.*;

class Solution {
    public int solution(String[] strArr) {
        int answer = 0;
        int[] nums = new int[100000];
        for(String str : strArr){
            nums[str.length()]++;
        }
        
        return Arrays.stream(nums).max().getAsInt();
    }
}